Activities/Java/UiPath.Java.Activities/JavaActivityWithParameters.cs

```diff
@@ -48,6 +48,17 @@ protected List<object> GetParameters(AsyncCodeActivityContext context)
             List<object> parameters = ParametersList.Get(context) ?? Parameters.Select(arg => arg.Get(context)).ToList();
             return parameters;
         }
-
+        protected List<Type> GetParameterTypes(AsyncCodeActivityContext context, List<object> parameters)
+        {
+            List<Type> parameterTypes = new List<Type>();
+            for(int index=0;index<Parameters.Count;index++)
+            {
+                if(null != parameters.ElementAt(index))
```

Comment:
parameter[index]. also spaces inside the for expression

Reply:
fixed